static/app/components/events/contexts/gpu/index.spec.tsx

```diff
@@ -33,43 +26,3 @@ export const gpuMetaMockData = {
     },
   },
 };
-
-const event = EventFixture({
-  _meta: {
-    contexts: {
-      gpu: gpuMetaMockData,
-    },
-  },
-});
-
-// Flaky test https://github.com/getsentry/sentry/actions/runs/4465585304/jobs/7842795315?pr=45984
-// eslint-disable-next-line
-describe.skip('gpu event context', function () {
```

Comment:
i think same here, i assume we'd like to fix this at some point otherwise this file is blank